src/task2/Main.java

```diff
@@ -0,0 +1,20 @@
+package task2;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        int sumOfWords = 0;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("How much lines u wanna have: ");
+        int quantityOfLines = sc.nextInt();
```

Comment:
what we would have in case if user would enter not a number? Or wrong format of number?

Reply:
Modified